frontend/device/pocketbook/device.lua

```diff
@@ -29,8 +29,9 @@ local PocketBook = Generic:new{
     hasKeys = yes,
     hasFrontlight = yes,
     canSuspend = no,
-    hasExitOptions = no,
-    canRestart = no,
+    hasExitOptions = yes,
+    canRestart = yes,
```

Comment:
hasExitOptions and canRestart return true on generic device, so there's no need to override them here (it shouldn't hurt either)